src/server/server_family.cc

```diff
@@ -118,19 +122,22 @@ string InferLoadFile(fs::path data_dir) {
   if (fs::exists(fl_path))
     return fl_path.generic_string();
 
+  SubstituteFilenameTsPlaceholder(&fl_path, "*");
   if (!fl_path.has_extension()) {
-    std::string glob = absl::StrCat(fl_path.generic_string(), "*");
-    io::Result<io::StatShortVec> short_vec = io::StatFiles(glob);
-
-    if (short_vec) {
-      auto it = std::find_if(short_vec->rbegin(), short_vec->rend(), [](const auto& stat) {
-        return absl::EndsWith(stat.name, ".rdb") || absl::EndsWith(stat.name, "summary.dfs");
-      });
-      if (it != short_vec->rend())
-        return it->name;
-    } else {
-      LOG(WARNING) << "Could not stat " << glob << ", error " << short_vec.error().message();
-    }
+    fl_path += ".*";
```

Comment:
you changed the logic here and I am not sure how this works..
If user has dbfilename=dump when using dragonfly save how willl adding .* to it will find the _summary.dfs?

Reply:
You are right, it didn't work :) I fixed the globbing and added more test coverage